_docs/other/discord-bot-usage.md

```diff
@@ -0,0 +1,51 @@
+---
+layout: page
+title: Discord Bot Usage
+permalink: /other/discord-bot-usage/
+---
+
+### Our discord bot provides some very useful toos to debug/issue track your server. We will go in-depth on how to use the bot commands and its tools.
```

Comment:
I don't recommend doing a header 3 as there's no header 2 above it, and will cause issues with screen readers. If we want bold, can we just do `*normal bold*` instead?